lib/context/uniformContext.ts

```diff
@@ -14,7 +15,7 @@ export function createUniformContext(serverContext?: NextPageContext) {
     transitionStore: new NextCookieTransitionDataStore({
       serverContext,
     }),
-    plugins: [enableContextDevTools()],
+    plugins: [enableContextDevTools(), enableGoogleGtagAnalytics()]
```

Comment:
I think we'd want to conditionally enable this plugin if a tag is defined, consider something like this

```
import { ContextPlugin } from '@uniformdev/context';
const plugins: ContextPlugins[] = [enableContextDevTools()];

if (process.env.GA_VARIABLE_NAME_HERE) {
    plugins.push(enableGoogleGtagAnalytics());
}
```

And then use this plugins array in the constructor of `Context`.